prql-compiler/src/ast/expr.rs

```diff
@@ -60,6 +60,46 @@ pub enum ExprKind {
 /// A name. Generally columns, tables, functions, variables.
 pub type Ident = String;
```

Comment:
I believe that this can be fully replaced by IdentWithNamespace. Because all idents can have namespace, right?

Reply:
Yes, my plan was to move it incrementally; then probably rename `IdentWithNamespace` back to `Ident`. I wanted to do the closest thing first, so we didn't get conflicts etc. (But maybe it's not that hard and we could do it in one fell swoop...)

Reply:
That's my philosophy on semantic branch - just do it in one fell swoop :D

Reply:
Ha! We are so aligned on so many dimensions; "average PR size" is the outlier!